app/src/js/reducers/workflows.js

```diff
@@ -3,19 +3,18 @@ import { set } from 'object-path';
 import {
   WORKFLOWS,
   WORKFLOWS_INFLIGHT,
-  WORKFLOWS_ERROR
+  WORKFLOWS_ERROR,
+  SEARCH_WORKFLOWS,
+  CLEAR_WORKFLOWS_SEARCH
 } from '../actions/types';
 import { createReducer } from '@reduxjs/toolkit';
 
 export const initialState = {
-  list: {
-    data: [],
-    meta: {},
-    params: {},
-    inflight: false,
-    error: false
-  },
-  map: {}
```

Comment:
What was this map here for? Nothing? Should we add it back?

Reply:
Good catch, it should be there.  It gets filled on the first call to the WORKFLOWS reducer

Reply:
7ef28d16